lib/light.js

```diff
@@ -1,32 +1,32 @@
-var Board = require("./board");
-var EVS = require("./evshield");
-var within = require("./mixins/within");
-var Fn = require("./fn");
-var Emitter = require("events").EventEmitter;
-var util = require("util");
-var priv = new Map();
-// var int16 = Fn.int16;
-var uint16 = Fn.uint16;
-var toFixed = Fn.toFixed;
-
-var Controllers = {
+const Board = require("./board");
+const EVS = require("./evshield");
+const within = require("./mixins/within");
+const Fn = require("./fn");
+const Emitter = require("events").EventEmitter;
+const util = require("util");
+const priv = new Map();
```

Comment:
Can a Map be a const?

Reply:
Yes, because you're not re-assigning the variable when updating the Map, same with Arrays and using `push` or `pop`.

Reply:
TIL ⬆️